src/components/Proposal/Stakes/index.tsx

```diff
@@ -0,0 +1,329 @@
+import { useState } from 'react';
+import moment from 'moment';
+import { useLocation } from 'react-router-dom';
+
+import { useContext } from 'contexts';
+import {
+  bnum,
+  calculateStakes,
+  formatBalance,
+  denormalizeBalance,
+} from 'utils';
+
+import { FiThumbsUp, FiThumbsDown } from 'react-icons/fi';
+
+import {
+  Question,
+  AmountBadge,
+  BlockchainLink,
+  HorizontalSeparator,
+} from 'components/common';
+
+import {
+  ActionArea,
+  SidebarRow,
+  Vote,
+  AmountInput,
+  PositiveSummary,
+  NegativeSummary,
+  SummaryDetails,
+  SummaryTotal,
+  ActionButton,
+} from '../styles';
+
+const Stakes = () => {
+  const {
+    context: { daoStore, configStore, providerStore, daoService, userStore },
+  } = useContext();
+
+  const [stakeAmount, setStakeAmount] = useState(0);
+  // We should get the ID in another way
+  const proposalId = useLocation().pathname.split('/')[3];
+  console.log('Proposal ID in Vote: ', proposalId);
+  const proposal = daoStore.getProposal(proposalId);
+  const proposalEvents = daoStore.getProposalEvents(proposalId);
+  const { account } = providerStore.getActiveWeb3React();
+  const scheme = daoStore.getScheme(proposal.scheme);
+
+  let stakedAmount = bnum(0);
+  let positiveStakesCount = proposalEvents.stakes.filter(
+    stake => stake.vote.toString() === '1'
+  ).length;
+  let negativeStakesCount = proposalEvents.stakes.filter(
+    stake => stake.vote.toString() === '2'
+  ).length;
```

Comment:
I think we should talk to @AugustoL to find out why stakes.vote === 1 is positive and 2 is negative and if we can change this for clarity

Reply:
The voting machine was design to have multiple options, but we use two by default, yes and not, 1 and 2, we should compare this against an enum like `Vote.YES` instead of `"1"`. Something to be added in the future.